### DISCUSSION TITLE
Links is not working with react-router-dom

### DISCUSSION THREAD

**Discussion by tareq96** [heroui-inc/heroui#3019] [Help]
Hi guys

I created this issue before and tried the suggested temp fix but it didn't work with me. Has anyone faced this issue before and have a fix or suggestion?

https://github.com/nextui-org/nextui/issues/2963

**Comment by wingkwong** [comment]
may take a look at https://github.com/nextui-org/nextui/issues/2934#issuecomment-2096289017

**Reply by tareq96** [reply]
Using the above fix make the links works but with a hard reload not soft navigate. So, this issue will be fixed for react-router-dom on 20th May too?

**Reply by wingkwong** [reply]
do you see the same error as reported in above issue? if not, please share your case so that we could take a look if this is another issue.

**Reply by tareq96** [reply]
There is no console erorr in my case, the issue I'm facing related to how the link work, so it does a full reload not a soft one.
